pkgs/top-level/all-packages.nix

```diff
@@ -4066,6 +4066,8 @@ in
 
   trousers = callPackage ../tools/security/trousers { };
 
+  tryton = pythonPackages.tryton;
```

Comment:
As this is not a library, please put it into a separate directory instead of `pythonPackages`